lib/structures/Guild.js

```diff
@@ -678,37 +663,33 @@ class Guild extends Base {
     }
 
     /**
-    * Get the ban list of the guild
-    * @returns {Promise<Object[]>} Resolves with an array of {reason: String, user: User}
+    * Leaves the voice channel in this guild
     */
-    getBans() {
-        return this._client.getGuildBans.call(this._client, this.id);
+    leaveVoiceChannel() {
+        this._client.closeVoiceConnection.call(this._client, this.id);
     }
 
     /**
-    * Get a ban from the ban list of a guild
-    * @arg {String} userID The ID of the banned user
-    * @returns {Promise<Object>} Resolves with {reason: String, user: User}
+    * Begin pruning the guild
+    * @arg {Number} [options] The options to pass to prune members
+    * @arg {Number} [options.days=7] The number of days of inactivity to prune for
+    * @arg {Array<String>} [options.includeRoles] An array of role IDs that members must have to be considered for pruning
+    * @arg {String} [options.reason] The reason to be displayed in audit logs
+    * @returns {Promise<Number>} Resolves with the number of pruned members
     */
-    getBan(userID) {
-        return this._client.getGuildBan.call(this._client, this.id, userID);
+    pruneMembers(options) {
```

Comment:
Missing `computePruneCount`

Reply:
do you mean getPruneCount?

Reply:
nope, this is supposed to be consistent with pruneMembers in client class

Reply:
Don't think it was there originally. I added it in